pkg/deploy/identity-provider/deployment_keycloak.go

```diff
@@ -221,6 +223,10 @@ func getSpecKeycloakDeployment(
 	}
 
 	keycloakEnv := []corev1.EnvVar{
+		{
+			Name:  "CM_REVISIONS",
```

Comment:
Why do we need this in the Keycloak and Che containers env? If we need to mark something, isn't labels does the work?

Reply:
Moved.